cmd/common-main.go

```diff
@@ -169,10 +172,21 @@ func handleCommonEnvVars() {
 		globalIsEnvDomainName = true
 	}
 
-	globalDomainIP = os.Getenv("MINIO_PUBLIC_IP")
-	if globalDomainName != "" && globalDomainIP != "" && globalEtcdClient != nil {
+	minioEndpointsEnv, ok := os.LookupEnv("MINIO_PUBLIC_IPS")
+	if ok {
+		minioEndpoints := strings.Split(minioEndpointsEnv, ",")
+		globalDomainIPs = set.NewStringSet()
+		for _, ip := range minioEndpoints {
+			if !s3utils.IsValidIP(ip) {
```

Comment:
Don't use this you can use pkg/net/ package for these things instead of s3utils.